### ISSUE TITLE
Choosing quality not working

### ISSUE THREAD

**Issue by AxeldeWater** [sdaqo/anipy-cli#155]
**Describe the bug**
When trying to use the -q or --quality options, it does not use that quality. It says it uses "hsl" quality, no idea what that means

**To Reproduce**
1. anipy-cli -q 720 or anipy-cli --quality 720
2. Choose the episode you want to watch
3. not the requested quality, shows "hsl"

**Expected behavior**
To play it in the set quality and show that

**Info**
Name: anipy_cli
Version: 2.7.31
Summary: Little tool in python to watch anime from the terminal (the better way to watch anime)
Home-page: https://github.com/sdaqo/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: /home/[~~~~]/.local/lib/python3.10/site-packages
Requires: beautifulsoup4, better-ffmpeg-progress, m3u8, moviepy, pycryptodomex, pypresence, python-dateutil, python-mpv, pyyaml, requests, setuptools, tqdm, yaspin
Required-by: 

**Platform**
 - OS: Linux Mint Cinnamon 21.3

**Comment by AxeldeWater** [comment]
Will that command "update" the program? So that i only have the new version installed?

**Comment by sdaqo** [comment]
If you install it with pip I am not sure but with pipx you should uninstall first. In both cases I would uninstall with:`pip uninstall anipy-cli`

**Comment by sdaqo** [comment]
Also after installing the new version do `anipy-cli --refresh-config`

**Comment by AxeldeWater** [comment]
@sdaqo It works now! I encountered a small hick up though, I tried -q worst one time and then it started playing as expected in the right resolution, but it kept only buffering a couple seconds and then only buffering the next seconds after it played those seconds and that in repeat. As if it were many videos in succession instead of one video (it also showed a total video length of a couple seconds each time instead of a video length of around 20 minutes, which is usually shown even when the video is not fully loaded). Only encountered that one time though and was not able to reproduce. 

Also, for some reason discord rich presence stopped worked. Maybe this is because I also updated Discord a couple hours ago, or it could be because of the anipy-cli installation being done through pipx. Not a huge problem for me, but still.
